stdlib/src/lzma.rs

```diff
@@ -322,6 +327,25 @@ mod _lzma {
                     .map_err(|_| new_lzma_error("Failed to initialize encoder", vm))?)
             }
         }
+
+        fn init_alone(
+            preset: u32,
+            filter_specs: Option<Vec<PyObjectRef>>,
+            vm: &VirtualMachine,
+        ) -> PyResult<Stream> {
+            if let Some(_filter_specs) = filter_specs {
+                Err(new_lzma_error(
```

Comment:
Unlike other actual errors, probably NotImplementedError fits better. Not very important though.